src/app/loans/loans-account-stepper/loans-account-details-step/loans-account-details-step.component.scss

```
@@ -30,3 +30,60 @@ mat-divider {
+  text-transform: uppercase;
+  letter-spacing: 0.04em;
+  opacity: 0.6;
+  line-height: 1;
+}
+
+.meta-value {
+  font-size: 0.85rem;
+  font-weight: 500;
+}
+
+::ng-deep .loan-product-select-panel {
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)